packages/playback-core/src/index.ts

```diff
@@ -175,10 +175,16 @@ export const setupHls = (
 
   const canUseNative = !type || (mediaEl?.canPlayType(type) ?? true);
   const hlsSupported = Hls.isSupported();
+  // NOTE: Native HLS playback on Android for LL-HLS has been flaky, so we're prefering
```

Comment:
Has it been ok for regular HLS? I've always found it to be severely lacking if you're doing more than just hitting play. Or is the idea here to leave it up to user-choice, except when we know things are bad?

Reply:
it's been "reasonable" iirc. Since users can configure `preferMse` via a single attr/prop, I believe we're erring on the side of "use native unless there's strong reasons not to" whenever they don't set that prop.